Documentation/gettingstarted/aws-eni.rst

```diff
@@ -63,6 +63,33 @@ Deploy Cilium release via Helm:
    the security groups for pod ENIs are derived from the primary ENI
    (``eth0``).
 
+.. note::
+
+   For AWS managed cluster (EKS) Cilium in CNI mode uses by default interface (ENI)
+   starting from 1 for pods IP allocation so effectively excluding 1 ENI (index 0), 
+   hence reducing pods IP pool. This should be taken into account creating 
+   clusters / deploying Cilium. Without special K8s/EKS configuration pods may and will 
+   become unschedulable in some cases. 
```

Comment:
"In some cases" is quite vague. I think it would be more informative to spell out these cases so that the reader can decide whether they are concerned about these cases or not.